README.md

```diff
@@ -1,2 +1,142 @@
+| ------------- |:-------------:| -----:|
+|  Researching, designing, implementing, and managing software programs     | The ability to communicate complex procedures to other colleagues  | 
+|  Testing and evaluating new programs     |  Commercial and business awareness     | 
+| Identifying areas for modification in existing programs and subsequently developing these modifications |  Good communication skills – both written and verbal      | 
+
+---
+## Jobs in this field
+
+![image] (https://www.devprojournal.com/wp-content/uploads/2020/12/software-developer-trends-696x392.jpg)
+
+### [Entry level job (0-2 years of experience)] (ca.indeed.com/software-developer-$60,000-jobs?advn=5761763028664928&vjk=4630f9f0efb376b9):
+
+##### Database Software Developer - Softworks Group Inc.
+
+##### Job Description: In this position, you will be working directly with customers to configure and customize the software to address complex business scenarios.
+
+##### Additional Skills & Experience
+* Experience using Azure SQL and other Azure technologies
+* Experience using embedded Power BI
+* Fluency in French
+
+##### Salary: $50,000 - $75,000
+
+### [Average level job (2-5 years of experience)] (companyca.indeed.com/software-developer-$80,000-jobs?vjk=07cee7ccaf529937&advn=6078419695541597):
+
+##### Full Stack Software Developer for SaaS company
+
+##### Job Description: We are looking for a Full Stack Developer who will be responsible for writing code, testing, launching, and providing support for large development projects.
+
+##### Additional Skills & Experience 
+* Frameworks: React, Django, WordPress is a plus
+* Web Concepts: REST, JSON, XML, OOP, Dependency Injection, SOAP
+* Databases: MySQl, MongoDB, Elasticsearch
+* Experience with cloud systems is a plus
+
+##### Salary: $65,000 to $136,665
+
+### [High level job (5+ years of experience)] (ca.indeed.com/software-developer-$140,000-jobs?advn=7280950653058030&vjk=8b5c793cdd409917) :
+
+##### Senior Full Stack Software Developer Toronto
+
+##### Job Description: On behalf of our client we are looking for a Senior Full-Stack Software Engineer. We can't wait to tell you more about this exciting growth opportunity to work with a dynamic team of engineers who are writing high-quality code, building high-impact products, and finding elegant solutions for complex technical problems.
+
+
+##### Additional Skills & Experience 
+* Experience with RoR, Django, Gin or comparable web frameworks.
+* Experience with modern front-end technology (React, Redux, Typescript, similar).
+* Experience with provisioning, managing, and documenting infrastructure (AWS, Azure, GCP, Terraform).
+* Experience in a very fast-paced, startup environment with the ability to quickly get up to speed, iterate, and take on projects.\
+
+##### Salary: $140,000 to $170,000
+---
+
+## Standard Education
+
+![image] (https://www.macleans.ca/wp-content/uploads/2018/11/PROGRAM-RANKINGS-2019-EDUCATION-810x445.jpg)
```

Comment:
misformatted image. does not render.  do not put a space between [] and ().  Should be like this:
```
![image](https://www.macleans.ca/wp-content/uploads/2018/11/PROGRAM-RANKINGS-2019-EDUCATION-810x445.jpg)
```
which renders to ...
![image](https://www.macleans.ca/wp-content/uploads/2018/11/PROGRAM-RANKINGS-2019-EDUCATION-810x445.jpg)